### DISCUSSION TITLE
Bug – Lien YouTube incorrect

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1576] [Autres sujets]
Je me suis aperçu que le lien donné sur [la page Documentation](https://ma-cantine-1.gitbook.io/ma-cantine-egalim/webinaires-les-defis-de-ma-cantine#a-destination-des-eple-le-31-mai-2022) pour accéder à l'enregistrement vidéo du webinaire aux EPLE ne fonctionne plus : https://www.youtube.com/watch?v=NXKsTB74aWQ

Il faut le remplacer par le nouveau lien : https://www.youtube.com/watch?v=y7RDuvAmR3A

**Comment by gauthierandre** [comment]
Merci JB pour le retour, lien modifié sur gitbook
